Adyen.xcodeproj/project.pbxproj

```diff
@@ -7513,7 +7520,7 @@
 				ENABLE_PREVIEWS = YES;
 				FRAMEWORK_SEARCH_PATHS = "$(inherited)";
 				INFOPLIST_FILE = Demo/SwiftUI/Info.plist;
-				IPHONEOS_DEPLOYMENT_TARGET = 13.6;
+				IPHONEOS_DEPLOYMENT_TARGET = 14.0;
```

Comment:
FYI @neelSharma12 I bumped the SwiftUI deployment target to 14.0 to be able to use ProgressView